lib/build.gradle

```diff
@@ -73,6 +73,9 @@ dependencies {
     kapt "com.github.bumptech.glide:compiler:4.11.0"
 
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4"
+
+    testImplementation "junit:junit:4.13.2"
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.7.3"
```

Comment:
@oluiscabral shouldn't the version of the kotlinx-coroutines-test library be the same as that of kotlinx-coroutines-android?

Reply:
No, those are two independent packages/projects